raaghu-elements/src/rds-notification/rds-notification.tsx

```
@@ -5,122 +5,169 @@ import RdsButton from "../rds-button";
-
-const RdsNotification = (props : RdsNotificationProps) => {
-    return (
-        <Fragment>
-            {props.notifications.map((notification) => (
+const RdsNotification = (props: RdsNotificationProps) => {
+  return (
+    <Fragment>
+      {props.notifications.map((notification) => (
+        <div
+          key={notification.userNotificationId}
+          className={`notification-card layout-${props.layout} style-${props.style} type-${props.type}`}
```

ESLint

[error] Expected indentation of 20 spaces but found 10.

(indent)